Resources/bin/cmux-ios-sim-server.py

```
@@ -0,0 +1,757 @@
+        super().__init__(message)
+        self.args_list = args or []
+        self.returncode = returncode
+        self.stdout = stdout
+        self.stderr = stderr
+
+
+def log(message):
+    print(message, file=sys.stderr, flush=True)
+
+
+def run(args, cwd=None, check=True, input_text=None):
```

Ruff (0.15.17)

[warning] Boolean default positional argument in function definition

(FBT002)